src/pages/instances/TerminalPayloadForm.tsx

```diff
@@ -79,110 +74,128 @@ const TerminalPayloadForm: FC<Props> = ({ close, onFinish }) => {
     }
   };
 
+  const updateContentHeight = () => {
+    updateMaxHeight("content-wrapper", "p-modal__footer", 64, "max-height");
```

Comment:
Where does the 64 come from?
Is it for the modal padding? The modal has a padding of `1rem` top and bottom, so shouldn't we use `2rem` or read the modal padding interactively?

Reply:
Yes, currently this "magic" value was obtained empirically from manual tests at different resolutions, trying to find a `px` amount that allowed to keep the footer always visible without taking too much vertical space around the modal.
But the idea for a future improvement would be to make the `updateMaxHeight` function to receive an `additionalOffset` parameter that is not necessarily in the `px` unit, but rather a full "value+unit" string that you then pass to the `calc` function:
```
  const style = `${targetProperty}: calc(100vh - ${offset}px - ${additionalOffset})`;
```
So if you pass "2rem", and you calculate an "above+below" offset of 20px, you'd have: `calc(100vh - 20px - 2rem)`.